src/components/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { ProgressBar } from "./ProgressBar";
+
+export default {
+  title: "ProgressBar|ProgressBar",
+  component: ProgressBar,
+  parameters: {
+    info: { inline: true, header: false },
+  },
+};
+
+export const Default = () => <ProgressBar step={1} />;
```

Comment:
suggestion to make this more generic, the param can be 

currentStep: number
totalSteps: number
descriptions: string[]

But i prefer not to do it now since we might be pre-optimising, just gonna leave this here in case we gonna use it elsewhere.

Reply:
hmm ya understand, was thinking about it when building the component. i think i'll add the total steps. for the description will just leave it for now.

Reply:
on second thought i'll not add the total steps, just leave them here for future refactoring. Cos dont wanna introduce more possibility for bugs.